flow2ml/Flow.py

```diff
@@ -202,3 +204,23 @@ def getDataset(self,img_dimensions,train_val_split):
     self.update_progress( 100/100.0,"Created Datasets" ) 
 
     return self.dataset
+
+  def deployTensorflowModels(self,conversions,model):
+    '''Deploy conversion from tensorflow model to tensorflowjs or tflite model'''
+
+    if(conversions['tfjs'] == conversions['tflite']):
```

Comment:
What if user wants model in both formats, then they provide ‘True’ to both conversions. So please remove this condition.

Reply:
> What if user wants model in both formats, then they provide ‘True’ to both conversions. So please remove this condition.

At one time ,only one conversion will happen.
Either tfjs or tflite.
As told yesterday by @yvkrishna .